src/viewer/upload/FileUpload.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { Alert, Button, Form, ProgressBar } from "react-bootstrap"
+import axios from 'axios';
+
+function UploadAlert(message) {
```

Comment:
I've done the same thing here that I did in the previous PR, which I'll correct when I do the DICOM uploader